content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning.md

```diff
@@ -564,12 +564,15 @@ spec:
     # the Webhook strategy instructs the API server to call an external webhook for any conversion between custom resources.
     strategy: Webhook
     # webhookClientConfig is required when strategy is `Webhook` and it configures the webhook endpoint to be called by API server.
+    # Ensuring compatibility with v1 and v1beta1
+    conversionReviewVersions: ["v1", "v1beta1"]
     webhookClientConfig:
```

Comment:
line 566 should appear before 569, right?

Reply:
Thank you for pointing that out, @tengqm! You're absolutely right—line 566 (the comment about compatibility with v1 and v1beta1) should appear before line 569 (the conversionReviewVersions field). I'll go ahead and correct the order to ensure the comment clearly describes the relevant field.